.travis.yml

```diff
@@ -3,5 +3,8 @@ language: ruby
 cache: bundler
 script:
   - docker-compose -f travis-docker-compose.yml run web tox
+  - docker-compose -f travis-docker-compose.yml run watch npm install
```

Comment:
FYI We aren't setting `NODE_ENV=development` anymore here or below. I suppose it doesn't matter for tests though, it may even be preferable not to set it